src/test/java/org/aszjch/demoapp/config/MinIOProperties.java

```diff
@@ -0,0 +1,7 @@
+package org.aszjch.demoapp.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "minio")
+public record MinIOProperties(String url, String username, String password) {
```

Comment:
spróbuj package scope

Reply:
WSZĘDZIE!

Reply:
Na razie wywaliłem sobie public z templatki klasy w IJ. Kolejne zmiany robię starając się jak najmniej public robić. W aktualnym kodzie nie zmieniam na razie tego jak to jest, bo już próbowałem tu w propertiesach i połowa spięcia minio na ServiceConnection na czerwono zaświeciła